src/redux/reducers/__tests__/socialReducer.spec.js

```diff
@@ -1,7 +1,7 @@
 /* eslint-disable import/no-duplicates */
-import { socialLoginSuccess, socialLoginFailure } from '@Redux/actions/socialActions';
-import { initialState } from '@Redux/reducers/socialReducer';
-import socialReducer from '@Redux/reducers/socialReducer';
+import { socialLoginSuccess } from '@Redux/actions/authActions';
+import { initialState } from '@Redux/reducers/authReducer';
+import authReducer from '@Redux/reducers/authReducer';
```

Comment:
Please take out the `eslint-disable` statement and fix the duplicate import error